### ISSUE TITLE
not properly shutting down async tasks

### ISSUE THREAD

**Issue by andrewkm** [filoghost/HolographicDisplays#286]
Paper 282

```
[15:30:48] [Server thread/ERROR]: Nag author: 'filoghost' of 'HolographicDisplays' about the following: This plugin is not properly shutting down its async tasks when it is being shut down. This task may throw errors during the final shutdown logs and might not complete before process dies.
[15:30:48] [Server thread/ERROR]: Nag author: 'filoghost' of 'HolographicDisplays' about the following: This plugin is not properly shutting down its async tasks when it is being shut down. This task may throw errors during the final shutdown logs and might not complete before process dies.
[15:30:48] [Server thread/ERROR]: Nag author: 'filoghost' of 'HolographicDisplays' about the following: This plugin is not properly shutting down its async tasks when it is being shut down. This task may throw errors during the final shutdown logs and might not complete before process dies.
[15:30:48] [Server thread/ERROR]: Nag author: 'filoghost' of 'HolographicDisplays' about the following: This plugin is not properly shutting down its async tasks when it is being shut down. This task may throw errors during the final shutdown logs and might not complete before process dies.
[15:30:48] [Server thread/ERROR]: Nag author: 'filoghost' of 'HolographicDisplays' about the following: This plugin is not properly shutting down its async tasks when it is being shut down. This task may throw errors during the final shutdown logs and might not complete before process dies.
```

**Comment by andrewkm** [comment]
@filoghost ever planning on looking into this?
It's some serious spam on every shutdown.

**Comment by andrewkm** [comment]
@filoghost please fix this spam.

**Comment by Kalebchoi** [comment]
Still an active issue @filoghost 
Please fix

**Comment by andrewkm** [comment]
@filoghost this is still happening, can you please fix this.

**Comment by TheRealRyGuy** [comment]
fixed in #512

**Comment by andrewkm** [comment]
https://github.com/filoghost/HolographicDisplays/issues/507
